planetaryimage/image.py

```diff
@@ -274,7 +279,7 @@ def _parse_data(self, stream):
             data_stream.close()
 
     def _parse_band_sequential_data(self, stream):
-        data = numpy.fromfile(stream, self.dtype, self.size)
+        data = numpy.fromstring(stream.read(self.size*4), self.dtype)
```

Comment:
This going to cause an extra copy and double the memory for all cases which can be significant for large images. I suggest we default to `numpy.fromfile` and fallback to `numpy.fromstring`, something like:

    try:
        data = numpy.fromfile(stream, self.dtype, self.size)
    except IOError:
        data = numpy.fromstring(stream.read(self.size*4), self.dtype)